Bottom: 1c452145c58e5771645421b6d9e5aaa3f39ac7b7
Top:    75136d30a288bb2a55aebf5e2cc22728ae0d27b5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-29 23:07:57 +0100

Refresh of hid-gtk-make-layer-selector-sh

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 4f25e72..0f936c9 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -844,6 +844,8 @@ make_layer_buttons (GtkWidget *layersel)
   gchar *color_string;
   gboolean active = TRUE;
 
+  /* XXX: I want these in group order! */
+
   for (i = 0; i < max_copper_layer; ++i)
     {
       layer_process (&color_string, &text, &active, i);
